### PR TITLE
Rewrite the introduction to shorten it and defer more to the BT spec.

### LINKED ISSUE #91: Clarify definition of Included Services.
I believe there is a typo in the third sentence of the third paragraph of the spec's [introduction](https://webbluetoothcg.github.io/web-bluetooth/#introduction):

`A Service exposes a collection of included Services and Characteristics.`

As far as I know a `Service` doesn't expose other `Services`. I believe it should read: `A Server exposes...`

### PR DESCRIPTION
**Pull request by jyasskin**
Demo at https://rawgit.com/jyasskin/web-bluetooth-1/rewrite-intro/index.html.

The description of the GATT hierarchy moved to the Information Model section.

Fixes #91 by describing included services in more detail.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1204,24 +1204,54 @@
...
+            and a collection of <a>Descriptor</a>s.
+            Depending on the properties of the Characteristic,
+            a <a>GATT Client</a> can read or write its value,
+            or register to be notified when the value changes.
+          </p>
+          <p>
+            Finally, a <a>Descriptor</a> contains a value (again an array of bytes)
+            that describes or configures its <a>Characteristic</a>.
+          </p>
+        </div>
```
Comment (scheib) [inline index.html]: I like this section, but would like easy ways to link to the Web Bluetooth API endpoints for the concepts. As a reader I'd mostly like to get from here to other parts of this spec, before getting to the underlying Bluetooth spec references. A few ideas:
- Add a final line in the note:
In Web Bluetooth the Bluetooth concepts are realized via: BluetoothDevice, BluetoothGATTRemoteServer, BluetoothGATTService, BluetoothGATTCharacteristic, BluetoothGATTDescriptor.
- Or change the links of all concepts for which we have objects to their object. E.g. "GATT Server" -> BluetoothGATTRemoteServer, which in turn references the Bluetooth spec GATT Server if necessary.

**Comment by scheib** [comment]
LGTM, left one comment re linking.

File: index.html (hunk 1 above)
Comment (jyasskin) [inline index.html]: Good point. I think I like your second option better. Then I'll have each relevant class say `X class represents a <a>Y Bluetooth concept</a>.`

**Comment by jyasskin** [comment]
See how the new linking looks. :)

File: index.html (hunk 1 above)
Comment (scheib) [inline index.html]: LGTM

**Pull request merged**